ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentActionMenu/DocumentActionMenu.js

```diff
@@ -43,14 +44,10 @@ export default class DocumentActionMenu extends Component {
     this.props.requestLoanForPatron(documentPid, patronPid, optionalParams);
   };
 
-  scrollTo(ref, menuItemName) {
-    ref.current.scrollIntoView(false, { behaviour: 'smooth', block: 'end' });
-    this.setState({ activeItem: menuItemName });
-  }
-
   render() {
-    const { document, relations, anchors } = this.props;
+    const { document, relations } = this.props;
     const { activeItem } = this.state;
+    const offset = -250;
```

Comment:
Since it doesn't change the `offset` variable doesn't need to be in the `render`, it can be declared as a class static or even a global const in the `invenioConfig` if it is or will be used elsewhere.

Reply:
I think I had that discussion already somewhere. It can't global because they are not always the same. but per record type it makes sense to have it in common